tools/lib/provision.py

```diff
@@ -219,6 +219,7 @@ def main(options):
     run(["sudo", "chown", "%s:%s" % (user_id, user_id), EMOJI_CACHE_PATH])
     run(["tools/setup/emoji/download-emoji-data"])
     run(["tools/setup/emoji/build_emoji"])
+    run(["tools/setup/build_pygment_data.py"])
```

Comment:
Also, need to add this to the similar section of `docs/install-generic-unix-dev.md`